### ISSUE TITLE
Parser breaks with a comment inside if condition

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#49]
Thank you for your patience while I submit more bizarre cases.

Sample code:
```java
public class MyTest {
    public void testMethod() throws Exception {
        String x;
        if (true
                // this is a sample comment
                || true)
            x = "foo";
    }
}
```

Stacktrace:
```
[error] MyTest.java: TypeError: Cannot create property 'comments' on string '||'
[error]     at SQLToAstVisitor.addComment (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3741:26)
[error]     at node.map.n (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3724:28)
[error]     at Array.map (<anonymous>)
[error]     at SQLToAstVisitor.addCommentsToAst (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3724:14)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3716:10)
[error]     at SQLToAstVisitor.operatorExpressionRest (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:2676:27)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain/lib/src/parse/cst/cst_visitor.js:52:42)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3714:29)
[error]     at SQLToAstVisitor.expression (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:2428:45)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain/lib/src/parse/cst/cst_visitor.js:52:42)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3714:29)
[error]     at SQLToAstVisitor.ifStatement (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:1768:28)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain/lib/src/parse/cst/cst_visitor.js:52:42)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:3714:29)
[error]     at SQLToAstVisitor.statementWithStartingToken (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain-java/src/visitor.js:1705:19)
[error]     at SQLToAstVisitor.visit (/home/grahamm/<snipped-project-dir>/node_modules/chevrotain/lib/src/parse/cst/cst_visitor.js:52:42)
```

**Comment by thorbenvh8** [comment]
Fixed in next release

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
